include/classes/Utility.class.php

```diff
@@ -317,7 +317,11 @@ public static function sanitizeIntArray($int_array) {
 		if (!is_array($int_array)) return false;
 		
 		$sanitized_array = array();
-		return $sanitized_array = array_filter($int_array, 'is_int');
+		foreach ($int_array as $i => $value) {

```

Comment:
Sorry about the misleading comment before. After some testing, this block can be replaced by `return array_filter($int_array, 'is_numeric');`. Note to use `is_numeric` instead of `is_int`. Thanks.